style.css

```diff
@@ -195,7 +195,7 @@ header p {
 .contact {
     margin: 0;
     padding: 0;
-    background-color: #222222;
+    background-color: #252826;
```

Comment:
Shows as white in IE10

Reply:
We couldn't resolve this, but decided that it wasn't worth fixing as the stite is still usable and readable without it.